hkpy/hkbase/observer/clients/__init__.py

```diff
@@ -0,0 +1,9 @@
+###
+# Copyright (c) 2019-present, IBM Research
+# Licensed under The MIT License [see LICENSE for details]
+###
+
+from .observerclient import ObserverClient as DefaultObserverClient
```

Comment:
I'd advise against this. Searching the code base becomes a lot harder with aliases.

Reply:
fixed